tests/test_app.py

```diff
@@ -80,8 +82,16 @@ async def mock_globus_id(*args, **kwargs):
 
 
 class AppClient:
```

Comment:
This needed some reworking with the now async `app_factory`. And it wasn't using the passed config anywhere.